src/ankit/coding_exam_4/PrintPalendromFromString.java

```diff
@@ -0,0 +1,41 @@
+package ankit.coding_exam_4;
+
+public class PrintPalendromFromString {
+	boolean isPalendrom(String word) {
+		String reverseWord = "";
+		for(int index =word.length()-1 ; index >= 0; index--) {
+			reverseWord = reverseWord + word.charAt(index);
+		}
+		if(reverseWord.equals(word)) {
+			return true;
+		}else 
+			return false;	
+	}
+
+	public String AllWordList(String str) {
+		String[] wordArray =  str.split(" ");
+		String output = "" ;
+		boolean flag = false;
+		for(int index = 0 ; index < wordArray.length; index++) {
+			flag = isPalendrom(wordArray[index]);	
+			if(flag==true) {
```

Comment:
boolean not require comparision as it already holding value true or false so if it's true then it will not goes to if part

Reply:
ok will not it down :)